components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAO.java

```diff
@@ -47,6 +47,11 @@ default AccessTokenDO getRefreshToken(String refreshToken) throws IdentityOAuth2
     void revokeOAuthConsentByApplicationAndUser(String username, String tenantDomain, String applicationName)
             throws IdentityOAuth2Exception;
 
+    default void revokeOAuthConsentsByApplication(String applicationName, String[] tenantDomain)
```

Comment:
Shall we add doc comment to the method even though the rest of the methods doesn't follow it.